src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,43 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
+import classnames from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  setModalOn: (state: boolean) => void;
+  currentTodo: Todo;
+  modalOn: boolean;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  modalOn,
+  setModalOn,
+  currentTodo,
+}) => {
+  const [currentUser, setCurrentUser] = useState<User>({
+    name: 'no user',
+    id: 0,
+    email: 'no email',
+    phone: 'no phone',
+  });
```

Comment:
```suggestion
  const [currentUser, setCurrentUser] = useState<User | null>(null);
```

Reply:
currentUser = Object is possibly 'null'.ts(2531)

Reply:
Right, need to update your code for it